src/screens/QRCodeScannerScreen/BarCodeMask.styles.ts

```diff
@@ -36,7 +36,6 @@ export default StyleSheet.create({
 		fontSize: 48,
 	},
 	textContainer: {
-		position: 'absolute',
```

Comment:
Removing this didn't change anything on my end -- the text remained in the same position. Is it the same for you @duskcloudxu ?